tasks/main.yml

```diff
@@ -14,6 +14,26 @@
     state: started
     enabled: yes
 
+- name: Copy Kibana SSL key
+  become: yes
+  copy:
+    content: "{{ kibana_server_ssl_key }}"
+    dest: "/etc/kibana/kibana_server_ssl.key"
+    owner: kibana
+    group: kibana
+    mode: 0600
+  notify: restart kibana
```

Comment:
I think these would need `when` conditions to pass with automated tests (or for people who don't have SSL enabled).

Also, I wonder if maybe we could just add support for the variables (e.g. in the templates/kibana.yml.j2 file) and leave the copying/placement up to the playbook itself. E.g. if you use Certbot, you wouldn't want to have this role copying certificates, since Certbot would get the LetsEncrypt certs for you...